hashcompute_test.go

```diff
@@ -46,6 +46,16 @@ func TestHashCompute(t *testing.T) {
 		{"_examples/sample1.jpg", "_examples/sample4.jpg", PerceptionHash, "PerceptionHash", 30},
 		{"_examples/sample2.jpg", "_examples/sample3.jpg", PerceptionHash, "PerceptionHash", 34},
 		{"_examples/sample2.jpg", "_examples/sample4.jpg", PerceptionHash, "PerceptionHash", 20},
+		{"_examples/sample1.jpg", "_examples/sample1.jpg", WaveletHash, "WaveletHash", 0},
+		{"_examples/sample2.jpg", "_examples/sample2.jpg", WaveletHash, "WaveletHash", 0},
+		{"_examples/sample3.jpg", "_examples/sample3.jpg", WaveletHash, "WaveletHash", 0},
+		{"_examples/sample4.jpg", "_examples/sample4.jpg", WaveletHash, "WaveletHash", 0},
+		{"_examples/sample1.jpg", "_examples/sample2.jpg", WaveletHash, "WaveletHash", 38},
+		{"_examples/sample1.jpg", "_examples/sample3.jpg", WaveletHash, "WaveletHash", 2},
+		{"_examples/sample1.jpg", "_examples/sample4.jpg", WaveletHash, "WaveletHash", 34},
+		{"_examples/sample2.jpg", "_examples/sample3.jpg", WaveletHash, "WaveletHash", 40},
+		{"_examples/sample2.jpg", "_examples/sample4.jpg", WaveletHash, "WaveletHash", 6},
+		{"_examples/sample3.jpg", "_examples/sample4.jpg", WaveletHash, "WaveletHash", 36},
```

Comment:
@PavelVGordeev 

Would you like to provide the result of [python version](https://pypi.org/project/ImageHash/)?